cmd/kubeadm/app/images/images.go

```diff
@@ -53,16 +53,19 @@ func GetEtcdImage(cfg *kubeadmapi.ClusterConfiguration) string {
 	return GetGenericImage(cfg.ImageRepository, constants.Etcd, etcdImageTag)
 }
 
+// GetPauseImage returns the image for the "pause" container
+func GetPauseImage(registry string) string {
```

Comment:
My vision for the funcs in the images files is for them to take `ClusterConfiguration` and then figure out from there how to generate the image string. This way we can keep future changes to the way we generate the images to this file only.
I know, that this will look like an overkill for `GetPauseImage`, but I think, that this may have benefits in the future and will keep things consistent with the rest of the bunch here.